<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color: rgb(255, 255, 255);
	background-color: rgb(83, 83, 83);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Downloads/Citibanamex_logo.svg.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>378</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(222, 11, 19);</string>
    </property>
    <property name="text">
     <string>Depósito Bancario Internacional</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>146</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>16</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(227, 11, 19);
font: 16pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string>Datos básicos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>76</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(18, 65, 125);
color: rgb(255, 255, 255);
font: 13pt &quot;Verdana&quot;;
</string>
    </property>
    <property name="text">
     <string>Nombre:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nombre">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 145, 180);
color: rgb(0, 0, 127);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>148</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(18, 65, 125);
color: rgb(255, 255, 255);
font: 13pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string>Apellido Paterno:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ap">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 145, 180);
color: rgb(0, 0, 127);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>130</y>
      <width>152</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(18, 65, 125);
font: 13pt &quot;Verdana&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Apellido Materno:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_am">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>160</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 145, 180);
color: rgb(0, 0, 127);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>50</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(18, 65, 125);
font: 13pt &quot;Verdana&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Sexo:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 145, 180);
font: 11pt &quot;Verdana&quot;;
color: rgb(0, 0, 127);</string>
    </property>
    <item>
     <property name="text">
      <string>Masculino</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Femenino</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>220</y>
      <width>186</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(18, 65, 125);
font: 13pt &quot;Verdana&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Fecha de nacimiento:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_fn">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>220</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 145, 180);

font: 10pt &quot;Verdana&quot;;
color: rgb(0, 0, 127);</string>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::ButtonSymbols::UpDownArrows</enum>
    </property>
    <property name="showGroupSeparator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="calendarPopup">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarioW">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>270</y>
      <width>261</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Verdana&quot;;
color: rgb(0, 0, 127);
background-color: rgb(83, 83, 83);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="gridVisible">
     <bool>false</bool>
    </property>
    <property name="horizontalHeaderFormat">
     <enum>QCalendarWidget::HorizontalHeaderFormat::ShortDayNames</enum>
    </property>
    <property name="verticalHeaderFormat">
     <enum>QCalendarWidget::VerticalHeaderFormat::NoVerticalHeader</enum>
    </property>
    <property name="navigationBarVisible">
     <bool>true</bool>
    </property>
    <property name="dateEditEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>511</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::HLine</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>185</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(18, 65, 125);
font: 13pt &quot;Verdana&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Lugar de nacimiento:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(18, 65, 125);
font: 13pt &quot;Verdana&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Estado:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>310</y>
      <width>89</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(18, 65, 125);
font: 13pt &quot;Verdana&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Municipio:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cB_Estados">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 145, 180);
font: 11pt &quot;Verdana&quot;;
color: rgb(0, 0, 127);</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cB_Municipios">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>340</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 145, 180);
font: 11pt &quot;Verdana&quot;;
color: rgb(0, 0, 127);</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>821</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::HLine</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>16</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(227, 11, 19);
font: 16pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string>Datos de deposito</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>174</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(18, 65, 125);
font: 13pt &quot;Verdana&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Tipo de documento:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cBTipoDoc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>191</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 145, 180);
font: 11pt &quot;Verdana&quot;;
color: rgb(0, 0, 127);</string>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>460</y>
      <width>166</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(18, 65, 125);
font: 13pt &quot;Verdana&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>No. de documento:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="linEdNumDoc">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>490</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 145, 180);
color: rgb(0, 0, 127);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>460</y>
      <width>135</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(18, 65, 125);
font: 13pt &quot;Verdana&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Motivo del giro:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cBMotivoGiro">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>490</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 145, 180);
font: 11pt &quot;Verdana&quot;;
color: rgb(0, 0, 127);</string>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>460</y>
      <width>146</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(18, 65, 125);
font: 13pt &quot;Verdana&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Monto (Dólares):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_monto">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>490</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 145, 180);
color: rgb(0, 0, 127);</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>540</y>
      <width>195</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(227, 11, 19);
font: 10pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string>Acepto politícas de datos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>530</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(18, 65, 125);
font: 10pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string>Guardar reporte</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Topicos/.venv/Lib/site-packages/PySide6/interfaces/logocitibanamex-removebg-preview.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dateEdit_fn</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>calendarioW</receiver>
   <slot>setSelectedDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>636</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calendarioW</sender>
   <signal>clicked(QDate)</signal>
   <receiver>dateEdit_fn</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>698</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>696</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
